java/src/com/google/template/soy/base/internal/SoyFileSupplier.java

```diff
@@ -138,7 +138,9 @@ public static SoyFileSupplier create(
       if (inputFileUrl.getProtocol().equals("file")) {
         // If the URL corresponds to a local file (such as a resource during local development),
         // open it up as a volatile file, so we can account for changes to the file.
-        return new VolatileSoyFileSupplier(new File(inputFileUrl.getPath()), soyFileKind);
+	// But first unescape space characters...
+        String filename = inputFileUrl.getFile().replace("%20", " ");
```

Comment:
Would converting to a URI and using getFile() instead of getRawFile() solve this problem more generally?  For example for path segments that %-escape other octets?